src/test/java/org/fcrepo/importexport/integration/ExporterIT.java

```diff
@@ -317,6 +321,44 @@ public void testExportCustomPredicatesInbound() throws Exception {
                 createResource(obj1.toString()),
                 createProperty("http://example.org/custom"),
                 createResource(col1.toString())));
+        assertFalse(collectionModel.contains(
+                createResource(other.toString()),
+                createProperty("http://example.org/other"),
+                createResource(col1.toString())));
+    }
+
+    @Test
+    public void testExportInboundFiles() throws Exception {
+        final UUID uuid = UUID.randomUUID();
+        final String baseURI = serverAddress + uuid;
+        final URI fileURI = URI.create(baseURI + "/file9");
+        createBody(fileURI, "This is some file content", "text/plain");
+
+        // export with inbound references
+        final Config config = new Config();
+        config.setMode("export");
+        config.setBaseDirectory(TARGET_DIR + "/" + uuid);
+        config.setResource(baseURI);
+        config.setRdfExtension(DEFAULT_RDF_EXT);
+        config.setRdfLanguage(DEFAULT_RDF_LANG);
+        config.setUsername(USERNAME);
+        config.setPassword(PASSWORD);
+        config.setPredicates(new String[]{ CONTAINS.toString() });
```

Comment:
This is the [default predicate](https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/main/java/org/fcrepo/importexport/common/Config.java#L49)... and can effectively be omitted.